--- Kbuild.include	2021-07-27 05:08:17.000000000 +1000
+++ Kbuild.include.fixed	2021-12-11 19:18:02.082926326 +1100
@@ -461,8 +461,8 @@
 # It's a common trick to declare makefile variable that contains space
 # we'll need it to convert the path string to list (string delimited by spaces)
 # and vice versa
-the-space :=
-the-space += 
+E =
+the-space = $E $E
 # TEGRA_ROOT_PATH is the relative path to the directory one level upper than $srctree
 _TEGRA_ROOT_PATH = $(subst ^$(realpath $(srctree)/..)/,,^$(realpath $(kbuild-dir)))
 # _TEGRA_REL_PATH is path like "../../../" that points to directory one level